tests/utils/command.go

```diff
@@ -57,35 +56,3 @@ func RegisterPingTest() {
 		gomega.Expect(healthy.Healthy).Should(gomega.BeTrue())
 	})
 }
-
-// RegisterNodeRun registers a before suite that starts an AvalancheGo process to use for the e2e tests
```

Comment:
We started using this `RegisterNodeRun` function here: https://github.com/ava-labs/precompile-evm/blob/main/tests/precompile/precompile_test.go#L18 to support builders that want to develop their own precompiles without maintaining a full fork of Subnet-EVM.

Could we add back this functionality, so that this PR doesn't break Precompile-EVM and then we can make a separate GitHub issue to leverage the same change to parallelize precompile e2e tests inside of Precompile-EVM as well?

I don't think we need to block this PR to figure out the best way to do that, so we should just make sure this doesn't break Precompile-EVM in the meantime.

Reply:
cc @ceyonur

Reply:
Great feedback and context. I redo the PR without touching this function and the final PR is much smaller.

Reply:
yea I agree. Optimistically, changes here should not affect current non-parallel tests and they should be able to run fine with changes here. I.e they should be running fine without `ginkgo run -procs=5`. If it's not possible, then we should have different functions for parallel testing, still compatible with sub-functions like running hardhat tests etc.